spring-auto-restdocs-core/src/main/java/capital/scalable/restdocs/response/MultipartContentOperationPreprocessor.java

```diff
@@ -0,0 +1,44 @@
+package capital.scalable.restdocs.response;
```

Comment:
The headers above the new files are missing but I can add them as well. We have a Maven plugin configured that adds them automatically.